docs/elasticsearch/index-templates.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: default
+title: Index Templates
+parent: Elasticsearch
+nav_order: 5
+---
+
+# Index template
+
+An index template formats all newly created indexes to a one like you may already have by automatically picking up on a naming schema.
```

Comment:
I think we can tune this opening sentence quite a bit. As is, it's pretty hard to parse. I'm not sure we need to mention the naming pattern until later. Something like, but not necessarily:

Index templates let you initialize new indices with predefined mappings and settings. For example, if you continuously index log data, you can define an index template so that all of these indices have the same number of shards and replicas.